sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/azure.yaml

```
@@ -8,10 +8,10 @@ provider:
       # These fix false positives:
       # C2y6y... Is the CosmosDB Local emulator
       # Eby8v... Is the Azure Storage / Azurite emulator
-      patternNot: "(\
-        EXAMPLE|\
-        Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==|\
-        C2y6yDjf5/R\\+ob0N8A7Cgv30VRDJIWEHLM\\+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==)"
+      patternNot:
+        - "EXAMPLE"
+        - "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw=="
+        - "C2y6yDjf5/R\\+ob0N8A7Cgv30VRDJIWEHLM\\+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw=="
```

```suggestion
        - "EXAMPLE"
        # Azure Storage / Azurite emulator
        - "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw=="
        # CosmosDB Local emulator
        - "C2y6yDjf5/R\\+ob0N8A7Cgv30VRDJIWEHLM\\+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw=="
```

Please also remove the comments on top of that 🙏